app/src/main/java/org/gem/indo/dooit/views/main/fragments/target/TargetFragment.java

```diff
@@ -315,4 +307,17 @@ public void run() {
             }
         });
     }
+
+    private void updateScrollImages() {
```

Comment:
@ChadGarrett Rename this method to something more fitting, like `updateNavButtons` or `updateNavCarets`. **"Images"** makes it seem like it has something to do with the Goal images.